home/screens/SignInScreen.tsx

```diff
@@ -135,8 +180,29 @@ class SignInView extends React.Component<Props, State> {
     this.setState({ password });
   };
 
+  _handleChangeOTP = (otp: string) => {
+    this.setState({ otp });
+  };
+
+  _handleSelectSMSSecondFactorDevice = async (device: SecondFactorDevice) => {
+    this.setState({ isLoading: true });
```

Comment:
Should we clear the OTP text field too? What do you think is a better UX?

Reply:
Thats a good idea!